public/app/features/plugins/admin/components/VersionInstallButton.tsx

```diff
@@ -174,7 +174,7 @@ function getButtonHiddenState(installState: PluginStatus, isPreinstalled: { foun
 
   // Handle downgrade case
   if (installState === PluginStatus.DOWNGRADE) {
-    return isPreinstalled.found && Boolean(config.featureToggles.preinstallAutoUpdate);
+    return isPreinstalled.found && Boolean(config.pluginCatalogPreinstalledPluginsAutoUpdate);
```

Comment:
How does this config property make its way to the frontend? I thought they were all passed via [frontendsettings.go](https://github.com/grafana/grafana/blob/preinstallAutoUpdate/pkg/api/frontendsettings.go#L307) but I don't see that file in changed files list in this PR? 🤔

Reply:
good catch, I missed testing this IRL 👍